.github/workflows/lint.yml

```diff
@@ -0,0 +1,15 @@
+name: Lint
+on: [push, pull_request]
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v0.1.7
```

Comment:
According to [the manual](https://github.com/golangci/golangci-lint-action), this should be `v1`

Reply:
@slaskawi we already been using it in Louketo with no issues. But if you would like to use `v1`, that's no problem from me.